ecep_cms/src/taccsite_cms/settings_custom.py

```diff
@@ -2,29 +2,30 @@
 # TACC WMA CMS SITE:
 # *.ECEP.TACC.UTEXAS.EDU
 
-# FAQ: Some _VARIABLES are duplicated from settings.py (but prefixed with "_")
-#      because current infrastructure lacks ability to reference default values
-
 ########################
-# DJANGO CMS SETTINGS
+# DJANGO_CMS
 ########################
 
 CMS_TEMPLATES = (
     ('standard.html', 'Standard'),
     ('fullwidth.html', 'Full Width'),
-
-    ('guide.html', 'Guide'),
-    ('guides/getting_started.html', 'Guide: Getting Started'),
-    ('guides/data_transfer.html', 'Guide: Data Transfer'),
-    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
-    ('guides/portal_technology.html', 'Guide: Portal Technology Stack')
 )
 
+########################
+# TACC: SEARCH
+########################
+
+# Support Google search instead of Portal's Elasticsearch
+SEARCH_PATH = '/site-search' # cuz Portal Nginx config hijacks /search
+SEARCH_QUERY_PARAM_NAME = 'q' # as Google expects
```

Comment:
**You added a feature in this PR?!** Nope. I moved the settings up from further below, to match Core-CMS oorder.